src/Symfony/Component/Notifier/Bridge/Lox24/README.md

```diff
@@ -0,0 +1,75 @@
+LOX24 SMS Notifier
+=========================
+
+Provides [LOX24 SMS Gateway](https://doc.lox24.eu/#tag/sms/operation/api_sms_post_collection) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+LOX24_DSN=lox24://TOKEN@default?from=FROM&type=SERVICE_CODE&voice_language=VOICE_LANGUAGE&is_text_delete=IS_TEXT_DELETE&callback_data=CALLBACK_DATA
+```
+
+where:
+
+- `TOKEN` (required) is LOX24 API v2 token.
+- `FROM` (required) is the sender of the message.
+- `TYPE` (optional) type of message: `sms` (by default) or `voice` (voice call).
+- `VOICE_LANGUAGE` (optional) if `type` is `voice`, then you can set the language of the voice message. Possible values: `de`, `en`, `es`, `fr`, `it` or `auto` (by default) per auto-detection.
+- `IS_TEXT_DELETE` (optional) delete SMS text from LOX24 database after sending SMS. Allowed values: `1` (true) or `0` (false). Default value: `0`.
+- `CALLBACK_DATA` (optional) additional data for the callback payload.
+
+See your account info at https://account.lox24.eu
+
+Send a Message
+---------------------------
+
+With a LOX24 Message, you can use the `LOX24Options` class to add
```

Comment:
I think this sentence needs to be removed

Reply:
done